pkg/registry/apis/provisioning/routes.go

```diff
@@ -26,7 +26,7 @@ func (b *APIBuilder) GetAPIRoutes() *builder.APIRoutes {
 					Get: &spec3.Operation{
 						OperationProps: spec3.OperationProps{
 							OperationId: "getResourceStats", // used for RTK client
-							Tags:        []string{"Provisioning"},
```

Comment:
Umm, the think is that `stats` is not about the repository but about resources and the same with settings. There are more generic and I guess that's why @ryantxu tagged them in that way.

Reply:
Are those used for anything on backend?

Reply:
The stats are across *all* provisioning sources -- this will also include things "provisioned" by terraform etc.  It will make more sense when this moves to something more like "managedBy"

Reply:
Another way is to add the 'Provisioning' tag to the delete endpoints, but I don't see that being done anywhere manually.